ui/apps/platform/src/Containers/Dashboard/Widgets/ComplianceLevelsByStandard.tsx

```diff
@@ -151,10 +144,7 @@ function ComplianceLevelsByStandard() {
     );
 
     const complianceData = useMemo(
-        () =>
-            processData(searchFilter, sortDataBy, data || previousData)
-                ?.slice(0, 6)
-                ?.reverse(), // Reverse since Victory charts renders items from bottom to top
+        () => processData(searchFilter, sortDataBy, data || previousData),
         [searchFilter, sortDataBy, data, previousData]
     );
```

Comment:
In retrospect, I have doubts that `useMemo` here provides a significant performance boost. It seems likely that most, if not all, rerenders would be due to a change in one of the memo dependencies here anyway, and the `n` in the processed data is around `10`, maybe `100` entries in extreme cases.